cloud_functions/titiler_cogs/titiler_cogs/app.py

```diff
@@ -28,6 +29,12 @@
 
 cog = TilerFactory()
 app.include_router(cog.router, tags=["Cloud Optimized GeoTIFF"])
+app.add_middleware(
+        CacheControlMiddleware,
+        cachecontrol="public",
```

Comment:
you probably want to set something lke "public, maxage=<some int>" https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/Cache-Control